analyzers/tests/SonarAnalyzer.UnitTest/Helpers/HashCodeTest.cs

```diff
@@ -43,17 +43,6 @@ public void Combine_ProducesDifferentResults(string input)
             hash5.Should().NotBe(0).And.NotBe(hash2).And.NotBe(hash3).And.NotBe(hash4);
         }
 
-        [TestMethod]
-        public void DictionaryContentHash_StableForUnsortedDictionary()
```

Comment:
Why was this removed?

Reply:
This was for `System.Collection.Generic.Dictionary<,>` which isn't supported anymore. I tried to support both types but it wasn't possible with one implementation and so I removed the test (I even tried `HashCode.DictionaryContentHash<TDict, TKey, TValue>(TDict dict) where TDict: IDictionary<TKey, TValue>` but this still used `IDictionary.GetEnumerator()` for the call `DictionaryContentHash<ImmutableDictionary<int, int>, int, int>()` under the hood).